src/content/reference/cli.mdx

```diff
@@ -77,9 +77,44 @@ The following flags can be used (very similar to `docker build`):
 | _--secret_       |  (list)  | Secret files exposed to the build. Format: `id=mysecret,src=/local/secret` |
 | _--export-cache_ | (string) | Image tag for exported cache when build (optional)                         |
 | _--cache-from_   |  (list)  | List of cache source images (optional)                                     |
+| _--platform_     | (string) | Specify which platform to build the container image for (optional)                                     |
+
 
 >  You can also use the `-f` to point to a Dockerfile. In this mode, `okteto build` will ignore your Okteto manifest, and directly build the image defined in the Dockerfile. Use this to build images that are not defined on your [Okteto manifest](https://www.okteto.com/docs/reference/manifest/).
 
+With `oktet build --platform` you can specify the platform (or architecture) for which you'd like to build the container images for. Suppose you want to deploy your web application on a Kubernetes cluster that consists of nodes running on both x86-64 and ARMv7 architectures. By using the multiplatform images
+built using `okteto build --platform`, you can deploy the same image across the cluster without worrying about the underlying hardware differences. 
+
+Let's consider an example where you have a Node.js application that you want to build and deploy on both x86_64 and ARM-based platforms. You have a Dockerfile in your project directory that defines the build process. Here's
+how Okteto CLI can help you build multiplatform images for your application:
+
+1. Building the image for x86_64 architecture:
+
+```console
+okteto build -t myapp:latest --platform linux/amd64
+```
+
+2. Building the image for ARMv7 architecture:
+
+```console
+okteto build -t myapp:latest --platform linux/arm/v7
+```
+
+3. Building a multiarchitectural image:
+
+```console
+okteto build -t myapp:latest --platform linux/amd64,linux/arm/v7
+```
+
+This command builds a multi-architecture Docker image named *myapp* with the latest tag for both x86_64 and ARM platforms.
+
+By using these commands, you can easily build the application image for different platforms without needing to maintain separate Dockerfiles or perform manual modifications. 
+This is particularly useful when you want to deploy your application to heterogeneous environments where you have both x86_64 and ARM-based devices, such as a mixed-cluster Kubernetes setup.
+
+
+Furthermore, This use case allows you to seamlessly distribute and scale your application across different hardware architectures without needing to maintain separate 
+images for each platform. 
```

Comment:
I think this is redundant with the previous paragraph. Let's keep the other one instead. 
```suggestion

```

Reply:
Fixed!